src/styles/blocks/menu.scss

```diff
@@ -0,0 +1,89 @@
+.menu {
+  background-color: $colorRed;
+
+  &__content {
+    @include content;
+
+    @include onTablet {
+      padding-top: 32px;
+    }
+  }
+
+  &__top {
+    @include topActions;
+
+    @include onDesktop {
+      --icon-width: #{$iconBig};
+      column-gap: 63px;
+    }
+
+    margin-bottom: 40px;
+
+    @include onDesktop {
+      margin-bottom: 55px;
+    }
+  }
+
+  &__list {
+    padding: 0;
+    margin: 0 0 23px;
+    list-style: none;
+
+    @include onDesktop {
+      margin: 0 0 39px;
+    }
+  }
+
+  &__item {
+    margin-bottom: 23px;
+
+    &:last-child {
+      margin-bottom: 0;
+    }
+  }
+
+  &__link {
+    display: inline-block;
+    padding-bottom: 8px;
+    border-bottom: 1px solid transparent;
+    font-weight: 700;
+    font-size: 22px;
+    line-height: 100%;
```

Comment:
It is recommended to use a unit for line-height to avoid potential scaling issues, e.g., line-height: 1.2;